# creates an openrave session
int32 sessionid
int32 options
---
int32 sessionid
